### PR TITLE
Add convenience method for fetching a Projects most recently added assets

### PR DESCRIPTION
**Pull request by strombergdev**

### CODE REVIEW

File: frameioclient/client.py [hunk 1]
```diff
@@ -219,6 +219,30 @@ def get_asset_children(self, asset_id, **kwargs):
     endpoint = '/assets/{}/children'.format(asset_id)
     return self._api_call('get', endpoint, kwargs)
 
+  def get_newest_assets(self, account_id, project_id, page_size):
+    """
+    Get a projects most recently added assets.
```
Comment (jhodges10) [inline frameioclient/client.py:224]: `project's` <- just being pedantic here. 

This is a great add! Thanks man :)

Comment (strombergdev) [reply frameioclient/client.py:224]: always mix those up :)

**Comment by strombergdev** [comment]
Realized this type of search only works well because auto-pagination isn't implemented in the current master, returning max 50 assets.

With working auto-pagination, this will fetch every asset in the project in sorted order with no upper limit. So working on updating [this PR](https://github.com/Frameio/python-frameio-client/pull/40) to include a limit on how many assets to fetch at most.

**Comment by jhodges10** [comment]
Looks good to me, just want to wait on the update to #40 before merging.

**Comment by strombergdev** [comment]
Last commit updates it to work with #56. So only merge if that one is implemented... 😄

**Pull request closed without merging**